python-package/lightgbm/engine.py

```diff
@@ -128,6 +128,7 @@ def train(params, train_set, num_boost_round=100,
     keep_training_booster : bool, optional (default=False)
         Whether the returned Booster will be used to keep training.
         If False, the returned value will be converted into _InnerPredictor before returning.
+        When your model is very large and cause the memory error, you can try to set this to ```True```, to avoid the model_to_string conversion.
```

Comment:
Just fix RST markdown and even more words about conversion.

```suggestion
        When your model is very large and cause the memory error, you can try to set this param to ``True`` to avoid the model conversion performed during the internal call of ``model_to_string``.
```